README.md

```diff
@@ -92,175 +134,66 @@ services:
+$ docker context use remote
+$ docker app run myrepo/hello:0.1.0
+``` 
 
 ## CNAB
 
-Under the hood `docker app` is [CNAB](https://cnab.io) compliant. It generates a
-CNAB from your application source and is able to install and manage any other
-CNAB too. CNAB specifies three actions which `docker app` provides as commands:
-* `install`
-* `upgrade`
-* `uninstall`
-
-Here is an example installing an Application Package, inspect it and
-then uninstalling it:
-```console
-$ docker app install examples/hello-world/example-hello-world.dockerapp --name hello
-Creating network hello_default
-Creating service hello_hello
-
-$ docker app inspect hello
-{
-    "Metadata": {
-        "version": "0.1.0",
-        "name": "hello-world",
-        "description": "Hello, World!",
-        "maintainers": [
-            {
-                "name": "user",
-                "email": "user@email.com"
-            }
-        ]
-    },
-    "Services": [
-        {
-            "Name": "hello",
-            "Image": "hashicorp/http-echo",
-            "Replicas": 1,
-            "Ports": "8080"
-        }
-    ],
-    "Parameters": {
-        "port": "8080",
-        "text": "Hello, World!"
-    },
-    "Attachments": [
-        {
-            "Path": "bundle.json",
-            "Size": 3189
-        }
-    ]
-}
-
-$ docker app uninstall hello
-Removing service hello_hello
-Removing network hello_default
-```
+Docker Apps are Docker’s implementation of the industry standard Cloud Native Application Bundle (CNAB). [CNAB](https://cnab.io/) is an industry specification put in place to facilitate the bundling, sharing, installing and managing of cloud-native apps that are not only made up of containers but also from such things as hosted databases, functions, etc. 
+Docker App is designed to abstract as many CNAB specifics as possible, to provide users with a tool that is easy to use while alleviating the need to bother with the CNAB specification.
 
 ## Installation
 
-**Note**: Docker app is a _command line_ plugin (not be confused with docker _engine_ plugins), extending the `docker` command with `app` sub-commands.
-It requires [Docker CLI](https://download.docker.com) 19.03.0 or later with experimental features enabled.
-Either set environment variable `DOCKER_CLI_EXPERIMENTAL=enabled`
-or update your [docker CLI configuration](https://docs.docker.com/engine/reference/commandline/cli/#experimental-features).
-
-**Note**: Docker-app can't be installed using the `docker plugin install` command (yet)
+Docker App is a *command line* plugin (not be confused with *docker engine plugins*) that extends the `docker` command with `app` sub-commands. It requires [Docker CLI](https://download.docker.com/) 19.03.0 or later, with experimental features enabled. Either set environment variable `DOCKER_CLI_EXPERIMENTAL=enabled` or update your [docker CLI configuration](https://docs.docker.com/engine/reference/commandline/cli/#experimental-features).
```

Comment:
Should we keep that experimental part?

Reply:
Yes Chris confirmed we need this for now.